.env.example

```
@@ -0,0 +1 @@
+ENV="production" # can be one of "production", "dev" or "testing"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)